firebase/firestore.rules

```diff
@@ -1,5 +1,17 @@
 rules_version = '2';
 
+function verifyMarkerDataComplete(data) {
+   return ('contentTitle' in data) &&
+          ('type' in data) &&
+          ('type' in data.type) &&
+          ('contact' in data) &&
+          ('loc' in data) &&
+          ('description' in data.loc) &&
+          ('lat' in data.loc) &&
```

Comment:
`lat` and `lng` should combined into a separate field in the document of the GeoPoint data type.
This is advised as we can perform geospatial queries later on.
for validation of the data type, we have already written a function with the name `verifyLatLngObject` within this `firestore.rules` file